zuul.d/jobs.yaml

```diff
@@ -781,7 +781,34 @@
     vars:
       ansible_collections_repo: github.com/ansible-collections/openvswitch.openvswitch
       ansible_test_command: network-integration
-      ansible_test_integration_targets: "openvswitch_.*"
```

Comment:
we should make this change in another PR, as ansible_test_integration_targets is going to be a large change to existing zuul jobs.